pephub/routers/api/v1/project.py

```diff
@@ -71,31 +72,31 @@ async def update_a_pep(
     """
     Update a PEP from a certain namespace
     """
-    # pull current project from database
-    peppy_project = db.get_project(namespace, project, tag=tag)
     raw_peppy_project = db.get_raw_project(namespace, project, tag=tag)
+    new_raw_project = raw_peppy_project.copy()
 
-    # update sample table
     if updated_project.sample_table_csv is not None:
-        # create pandas df from csv in memory
         sample_table_csv = StringIO(updated_project.sample_table_csv)
         sample_table_df = pd.read_csv(sample_table_csv)
-
-        # drop any empty columns
         sample_table_df = sample_table_df.dropna(axis=1, how="all")
+        sample_table_df_json = sample_table_df.to_dict()
+
+        new_raw_project["_sample_dict"] = sample_table_df_json
```

Comment:
Instead of hard-coding the key `_sample_dict`, can we import it from `peppy`? Something like:

```python
from peppy.const import SAMPLE_DICT_KEY
...
new_raw_project[SAMPLE_DICT_KEY] = sample_table_df_json
```

Reply:
Very good idea!